site/docs/tooltip.md

```diff
@@ -5,26 +5,30 @@ title: Tooltip
 permalink: /docs/tooltip.html
 ---
 
-Tooltip can provide details of a particular data point on demand. There are two ways to create a tooltip in Vega-Lite.
+Tooltip can provide details of a particular data point on demand. Tooltips are created with the `tooltip` channel. By default, the renderer with use native tooltips.
```

Comment:
> with

You mean "will"?

> use native tooltips.

"generate tooltips via native HTML "title" attribute." ?

Reply:
Done